.gitlab-ci.yml

```diff
@@ -251,7 +251,33 @@ build_live:
+    UNTRACKED_EXTRAS: "data,content/en/agent/basic_agent_usage/heroku.md,
+      content/en/agent/basic_agent_usage/ansible.md,
+      content/en/agent/basic_agent_usage/chef.md,
+      content/en/agent/basic_agent_usage/puppet.md,
+      content/en/developers/integrations,
+      content/en/agent/basic_agent_usage/saltstack.md,
+      content/en/developers/amazon_cloudformation.md,
+      content/en/integrations,
+      content/en/logs/log_collection/android.md,
+      content/en/logs/log_collection/ios.md,
+      content/en/tracing/setup_overview/setup/android.md,
+      content/en/tracing/setup_overview/setup/ruby.md,
+      content/en/security_platform/default_rules,
+      content/en/serverless/libraries_integrations/forwarder.md,
+      content/en/serverless/libraries_integrations/plugin.md,
+      content/en/serverless/libraries_integrations/macro.md,
+      content/en/serverless/libraries_integrations/cli.md,
+      content/en/real_user_monitoring/android/_index.md,
+      content/en/real_user_monitoring/android/data_collected.md,
+      content/en/real_user_monitoring/android/advanced_configuration.md,
+      content/en/real_user_monitoring/android/troubleshooting.md,
+      content/en/real_user_monitoring/android/integrated_libraries.md,
+      content/en/real_user_monitoring/error_tracking/android.md,
+      content/en/real_user_monitoring/browser/_index.md,
+      content/en/real_user_monitoring/ios.md,
+      content/en/real_user_monitoring/reactnative.md,
+      content/en/logs/log_collection/javascript.md"
```

Comment:
@apigirl So I tested and this change doesn't work with the current setup. However i worked on a PR here https://github.com/DataDog/documentation/pull/10459 so that we can reduce this and stop having to define generated files all the time.

Reply:
I completely missed this last week. You are the best David!!! Thanks for doing this.